resources/views/contests/partials/_form_contest.blade.php

```diff
@@ -4,12 +4,12 @@
 
 <div class="form-item -padded">
     <label class="field-label" for="signup_start_date">Signup Start Date:</label>
-    <input type="date" name="signup_start_date" id="signup_start_date" value={{ $contest->waitingRoom->signup_start_date or old($contest->waitingRoom->signup_start_date)}} class="text-field"></input>
+    <input type="date" name="signup_start_date" id="signup_start_date" value={{ $contest->waitingRoom->signup_start_date or 'MM/DD/YYYY' }} class="text-field"></input>
```

Comment:
Oh, it was used incorrectly! You only pass the string name of the item, not the actual property that might not exist. You can do this:

```php
<input type="date" name="signup_start_date" id="signup_start_date" value={{ $contest->waitingRoom->signup_start_date or old('signup_start_date', 'MM/DD/YYYY')}} class="text-field"></input>
```

If an old value exists, then it'll be in sitting in a property in the request object which holds an array of `old` data, and you grab it by passing a string of what it's called.